Assets/Scripts/Controllers/BuildModeController.cs

```diff
@@ -230,9 +230,11 @@ public void DoBuild(Tile t)
 
     public bool DoesBuildJobOverlapExistingBuildJob(Tile t, string furnitureType)
     {
-        for (int x_off = t.X; x_off < (t.X + WorldController.Instance.World.furniturePrototypes[furnitureType].Width); x_off++)
+        Furniture furniturePrototype = WorldController.Instance.World.furniturePrototypes[furnitureType];
+
+        for (int x_off = t.X; x_off < (t.X + furniturePrototype.Width); x_off++)
         {
-            for (int y_off = t.Y; y_off < (t.Y + WorldController.Instance.World.furniturePrototypes[furnitureType].Height); y_off++)
```

Comment:
Could put that as a variable on the line above ` int height = WorldController.Instance.World.furniturePrototypes[furnitureType].Height`? Not only is it more readable but think expression will be check once per iteration which is a waste unless the furniture height changes.